web_access_rule_buttons/models/models.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, api, exceptions
+
+
+class Base(models.AbstractModel):
+    """ The base model, which is implicitly inherited by all models. """
+    _inherit = 'base'
+
+    @api.multi
+    def check_access_rule_all(self, operations=None):
+        """Verifies that the operation given by ``operations`` is allowed for
+        the user according to ir.rules.
+
+        If ``operations`` is empty, it returns the result for all actions.
+
+       :param operation: a list of ``read``, ``create``, ``write``, ``unlink``
+       :return: {operation: access} (access is a boolean)
+        """
+        if operations or None:
```

Comment:
This is 'True' if operations is not empty.
```suggestion
        if not operations or not any(operations):
```